### PR TITLE
Issue 66 completed

### PR DESCRIPTION
**Pull request by remingtontr**
Article rewritten and completed.

### CODE REVIEW

File: docs/device-tree-overlay/software-spi.md [hunk 1]
```diff
@@ -2,57 +2,52 @@
...
+
 ## Source Code
+The DTS fragment that enables the SPI Bus functionality can be found in the [OnionIoT/OpenWRT-Packages GitHub repository](https://github.com/OnionIoT/OpenWRT-Packages/blob/openwrt-22.03/onion-dt-overlay/src/sw-spi.dts). 
 
-The DTS fragment that enables this functionality can be found at: https://github.com/OnionIoT/OpenWRT-Packages/blob/openwrt-22.03/onion-dt-overlay/src/sw-spi.dts
+The package definition can be found at: [https://github.com/OnionIoT/OpenWRT-Packages/blob/openwrt-22.03/onion-dt-overlay/Makefile](https://github.com/OnionIot/OpenWRT-Packages/blob/openwrt-22.03/onion-dt-overlay/Makefile).
 
 It is part of the `onion-dt-overlay` package.
 
-#
```
Comment (greenbreakfast) [inline docs/device-tree-overlay/software-spi.md:54]: Please add this back in to enable comments on the published article

File: docs/device-tree-overlay/software-spi.md [hunk 2]
```diff
@@ -2,57 +2,52 @@
...
-
-import { GiscusDocComment } from '/src/components/GiscusComment';
-
-<GiscusDocComment />
```
Comment (greenbreakfast) [inline docs/device-tree-overlay/software-spi.md:58]: Please add this back in to enable comments on the published article

**Comment by remingtontr** [comment]
Fixed and pushed.

Tom Remington
________________________________

**Comment by greenbreakfast** [comment]
👍 
cc #65

**Pull request merged**